src/views/NewLanding/index.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useEffect } from 'react';
+
+import {
+  Button,
+  Grid,
+  Column
+} from '@carbon/react';
+import { Login } from '@carbon/icons-react';
+import { KeycloakLoginOptions } from 'keycloak-js';
+import { useNavigate } from 'react-router-dom';
+import { useAuth } from '../../contexts/AuthContext';
+import BCGovLogo from '../../components/BCGovLogo';
+import Seeding from '../../assets/img/seeding.png';
+import LoginProviders from '../../types/LoginProviders';
+import './styles.css';
+import getUrlQueryParam from '../../utils/UrlUtils';
+
+const NewLanding = () => {
+  const { startKeycloak, login, signed } = useAuth();
+  const navigate = useNavigate();
+  const homePage = '/home';
+
+  const handleLogin = (provider: LoginProviders) => {
+    if (signed) {
+      navigate(getUrlQueryParam(window.location, 'page') || homePage);
+      return;
+    }
+
+    const idpHint = provider;
+    const loginOptions: KeycloakLoginOptions = {
+      redirectUri: `${window.location.origin}${homePage}`,
+      idpHint
+    };
+
+    login(loginOptions);
+  };
+
+  useEffect(() => {
+    if (signed) {
+      navigate(getUrlQueryParam(window.location, 'page') || homePage);
+    } else {
+      startKeycloak();
+    }
+  }, [signed]);
+
+  return (
+    <Grid fullWidth>
+      {/* First - Column */}
+      <Column lg={8}>
+        {/* Logo */}
+        <BCGovLogo />
+
+        {/* Welcome - Title and Subtitle */}
+        <h1 data-testid="landing-title" className="landing-title">Welcome to SPAR</h1>
+        <p data-testid="landing-subtitle" className="landing-subtitle">
+          Seed Planning and Registry Application
+        </p>
+
+        {/* Description */}
+        <h3 data-testid="landing-desc" className="landing-desc">
+          Register and storage your seed and meet your annual
+          reforestation needs using
+          <span className="spar-span"> SPAR</span>
+        </h3>
+
+        {/* Login buttons */}
+        <Button
+          onClick={() => { handleLogin(LoginProviders.IDIR); }}
+          size="md"
+          renderIcon={Login}
+          data-testid="landing-button__idir"
+          className="btn-landing"
+        >
+          Login with IDIR
+        </Button>
+        &nbsp;
+        <Button
+          kind="tertiary"
+          onClick={() => { handleLogin(LoginProviders.BCEID_BUSINESS); }}
+          size="md"
+          renderIcon={Login}
+          data-testid="landing-button__bceid"
+          className="btn-landing"
+        >
+          Login with Business BCeID
+        </Button>
+      </Column>
+
+      {/* Second - Column */}
+      <Column lg={5}>
+        <img src={Seeding} alt="Seeding" className="seeding-img" />
```

Comment:
Can/Should we elaborate on this alternative text? I don't know how elaborate it should be: maybe "Close on a small seedling recently planted on the dirt and watered, illuminated by the rays of the Sun at dawn." Maybe this is not it, but the image was chosen for some reason (because it gives a feeling of coziness, or hope, or resilience, or all of them, or something else), and I have the feeling the alternative text should wither try to convey that reason or give a more accurate, if dry, description of the image.

This looks like something the design team could help us with.

Reply:
Yes, I'll write a better text then we can start a conversation related to alt texts. Thanks man!